src/AiFactory.cpp

```diff
@@ -631,15 +638,16 @@ void AiFactory::AddDefaultNonCombatStrategies(Player* player, PlayerbotAI* const
     {
         nonCombatEngine->ChangeStrategy(sPlayerbotAIConfig->nonCombatStrategies);
     }
-    // nonCombatEngine->addStrategy("battleground");
-    // nonCombatEngine->addStrategy("warsong");
+
     // Battleground switch
     if (player->InBattleground() && player->GetBattleground())
     {
-        nonCombatEngine->addStrategies("nc", "chat", "default", "buff", "food", "mount", "pvp", "dps assist", "attack tagged", nullptr);
+        nonCombatEngine->addStrategies("racials", "nc", "chat", "default", "buff", "food", "mount", "collision", "pvp", "dps assist", "attack tagged", "emote", nullptr);
         nonCombatEngine->removeStrategy("custom::say");
         nonCombatEngine->removeStrategy("travel");
         nonCombatEngine->removeStrategy("rpg");
+        nonCombatEngine->removeStrategy("rpg craft");
+        nonCombatEngine->removeStrategy("follow");
```

Comment:
Don’t just change it for granted, you need to make sure the strategies are working correctly and explain what specific problem they solve.